webapp/src/utils/time.ts

```diff
@@ -39,3 +39,14 @@ export const formatYYYYMMddHHMMSS = (timestamp: number): string => {
   const seconds = String(date.getSeconds()).padStart(2, '0');
   return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
 }
+
+export const zonedDateFormatYYYYMMddHHMMSS = (utcDateTime: string): string => {
```

Comment:
Can we find a better name for this .. or maybe even some standard method that creates this output .. it is basically ISO date time format right?

Reply:
Also do we need a test for this?

Reply:
I have changed the function name.

Reply:
Thanks